src/schema/nmdc.yaml

```diff
@@ -307,6 +307,20 @@ classes:
     exact_mappings:
       - OBI:0000747
       - NCIT:C43412
+
+  credit record:
```

Comment:
call this study role

add class_uri: Prov: Association

see https://github.com/linkml/linkml/pull/293/files

Reply:
done. passes `Check Pull Request`